system/modules/comments/dca/tl_comments_notify.php

```diff
@@ -26,10 +26,11 @@
 			'keys' => array
 			(
 				'id' => 'primary',
-				'source' => 'index',
-				'parent' => 'index',
-				'tokenConfirm' => 'index',
-				'tokenRemove' => 'index'
+				'source' => 'index', // FIXME: remove?
+				'parent' => 'index', // FIXME: remove?
+				'tokenConfirm' => 'index', // FIXME: remove?
```

Comment:
As discussed on Mumble on March, 26th, we should remove the single indexes on `source`, `parent` and `tokenConfirm`.